user_guide/calendars_overview.rst

```diff
@@ -25,9 +25,9 @@ and the user, to whom the task/event has been assigned.
 System and Organization Calendars
 ---------------------------------
 
-Events in the System and Organization calendars can be viewed and processed by all the users within the 
-system/organization, unless these is restricted by their 
-:ref:`roles <user-guide-user-management-permissions>`.
+Events in the System and Organization calendars can be viewed by all the users within the system/organization. 
+The ability to create, edit, delete the events depends on their :ref:`roles <user-guide-user-management-permissions>`
+and :ref:`capabilities <admin-capabilities-org-calendar-events>`.
```

Comment:
Edit: The ability to create, edit, and delete events depends on the user's :ref:`roles <user-guide-user-management-permissions>` and :ref:`capabilities <admin-capabilities-org-calendar-events>`.